test/model-serialization.test.ts

```diff
@@ -22,7 +22,7 @@ function initFunc(plates: Record<number, Plate>) {
 let modelImgData: ImageData | null = null;
 
 beforeAll(done => {
-  (global as any).getModelImage("testModel.png", (imgData: ImageData) => {
+  (self as any).getModelImage("testModel.png", (imgData: ImageData) => {
```

Comment:
Having used the `(global as any)` trick in test code myself, I'm curious about the reason for this change. Is there some advantage I should be aware of?

Reply:
After I updated a bunch of dependencies, something was complaining about `global`. I don't remember now if it was ESlint or TS (or probably TS-Jest).